src/movies.js

```diff
@@ -1,22 +1,70 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+  let container = [];
```

Comment:
Just so you know – the container variable is not necessary here. The map method returns a new array, and the push method is not needed.
You can directly use the map method to return the array of directors, like this:

```
function getAllDirectors(moviesArray) {
  return moviesArray.map((movie) => movie.director);
}
```
This way, the function will return an array containing the 'director' property of each object in the input array without creating an additional container variable.